Makefile

```diff
@@ -34,11 +42,32 @@ GO_TEST_PARALLEL := $(shell echo $$(( $(NPROCS) / 2 )))
 
 GO_INTEGRATION_TESTS_SUBDIRS = test
 
+GO_STATIC_PACKAGES = $(GO_PROJECT)/cmd/oam-runtime
 GO_LDFLAGS += -X $(GO_PROJECT)/pkg/version.Version=$(VERSION)
-GO_SUBDIRS += pkg apis
+GO_SUBDIRS += cmd pkg apis
 GO111MODULE = on
 -include build/makelib/golang.mk
 
+# ====================================================================================
+# Setup Images
+# Due to the way that the shared build logic works, images should
+# all be in folders at the same level (no additional levels of nesting).
+
+DOCKER_REGISTRY = crossplane
+IMAGE_DIR=$(ROOT_DIR)
+IMAGE = $(BUILD_REGISTRY)/oam-runtime-$(ARCH)
```

Comment:
looks like the image name also contains (ARCH), I guess it's not very important in the sense that it's not user facing

Reply:
We will push multi-arch images, see https://hub.docker.com/r/crossplane/crossplane/tags for example of how each image has an AMD and an ARM flavor